### PR TITLE
Update workers.yaml

### PR DESCRIPTION
**Pull request by ohodson**
Add external release notes from edgeworker 2023.10.402 and 2023.10.1100.

### CODE REVIEW

File: data/changelogs/workers.yaml [hunk 1]
```diff
@@ -2,6 +2,13 @@
 link: "/workers/platform/changelog/"
 productName: Workers
 entries:
+- publish_date: '2023-10-12'
+  description: |-
+    - A Socket's WritableStream now ensures the connection has opened before resolving the promise returned by `close`.
+- publish_date: '2023-10-09'
+  description: |-
+    - The Web Platform standard `CustomEvent` class (https://dom.spec.whatwg.org/#interface-customevent) is now available in Workers.
+    - Fixed a bug in the WebCrypto API where the `publicExponent` field of the algorithm of RSA keys would have the wrong type. Use the cryptoPreservePublicExponent compatibility flag to enable the new behavior.
```
Comment (deadlypants1973) [inline data/changelogs/workers.yaml]:
```suggestion
    - Fixed a bug in the WebCrypto API where the `publicExponent` field of the algorithm of RSA keys would have the wrong type. Use the `cryptoPreservePublicExponent` compatibility flag to enable the new behavior.
```

**Review by deadlypants1973 — APPROVED** [review]

File: data/changelogs/workers.yaml [hunk 2]
```diff
@@ -6,6 +6,13 @@ entries:
   description: |-
     - The limit of 3 Cron Triggers per Worker has been removed. Account-level limits on the total number of
       Cron Triggers across all Workers still apply.
+- publish_date: '2023-10-12'
+  description: |-
+    - A Socket's WritableStream now ensures the connection has opened before resolving the promise returned by `close`.
+- publish_date: '2023-10-09'
+  description: |-
+    - The Web Platform standard `CustomEvent` class (https://dom.spec.whatwg.org/#interface-customevent) is now available in Workers.
```
Comment (irvinebroque) [inline data/changelogs/workers.yaml]:
```suggestion
    - The Web Platform standard [`CustomEvent` class](https://dom.spec.whatwg.org/#interface-customevent) is now available in Workers.
```

File: data/changelogs/workers.yaml [hunk 3]
```diff
@@ -6,6 +6,13 @@ entries:
...
+    - Fixed a bug in the WebCrypto API where the `publicExponent` field of the algorithm of RSA keys would have the wrong type. Use the `cryptoPreservePublicExponent` compatibility flag to enable the new behavior.
```
Comment (irvinebroque) [inline data/changelogs/workers.yaml:15]: Addressed in https://github.com/cloudflare/cloudflare-docs/pull/11371 cc @fhanau

File: data/changelogs/workers.yaml [hunk 4]
```diff
@@ -6,6 +6,13 @@ entries:
   description: |-
     - The limit of 3 Cron Triggers per Worker has been removed. Account-level limits on the total number of
       Cron Triggers across all Workers still apply.
+- publish_date: '2023-10-12'
+  description: |-
+    - A Socket's WritableStream now ensures the connection has opened before resolving the promise returned by `close`.
```
Comment (irvinebroque) [inline data/changelogs/workers.yaml]:
```suggestion
    - A [TCP Socket](/workers/runtime-apis/tcp-sockets/)'s WritableStream now ensures the connection has opened before resolving the promise returned by `close`.
```

**Review by irvinebroque — APPROVED** [review]
reuse content from https://github.com/cloudflare/cloudflare-docs/pull/11371/files#diff-4acbd6b8cf3c624e8c3e4c6c17991289cdbd5b4ee66c60ef78ed2f29b1297166R6-R9 for the crypto compat flag?

**Comment by ohodson** [comment]
Stuck with "Compiles Expected — Waiting for status to be reported".

Another attempt: https://github.com/cloudflare/cloudflare-docs/pull/11640

**Pull request closed without merging**